src/analyzer/plugins/analyzerkeyfinder.cpp

```diff
@@ -101,11 +101,6 @@ bool AnalyzerKeyFinder::processSamples(const CSAMPLE* pIn, const int iLen) {
         }
     }
     m_keyFinder.progressiveChromagram(m_audioData, m_workspace);
-    ChromaticKey key = chromaticKeyFromKeyFinderKeyT(
-            m_keyFinder.keyOfChromagram(m_workspace));
-    if (key != m_previousKey) {
-        m_resultKeys.push_back(qMakePair(key, m_currentFrame));
```

Comment:
It is OK to remove this, because this uses a bigger chunk of chromagram each call until it contains the full track. 
What we actually need in future is the a key map. 
I think we need a function that returns the keys per hop.